src/lib/xapi.ts

```diff
@@ -150,6 +150,7 @@ export const Verb = {
 	PlayedVideo: 'https://w3id.org/xapi/video/verbs/played',
 	Rated: 'http://id.tincanapi.com/verb/rated',
 	Registered: 'http://adlnet.gov/expapi/verbs/registered',
+	Retrieved: 'https://w3id.org/xapi/dod-isd/verbs/retrieved',
```

Comment:
What does retrieved mean?

Reply:
"Get or bring something back; regain possession of.* ;D

If an article is removed it is archived which changes it's state. Adding or removing to learning plan changes its preference. There doesn't seem to be a way of removing a state once changed? so we need a way of saying something iks no longer archived when it is readded once it is removed.